data/nlu/responses/responses_medicine.yml

```diff
@@ -1,26 +1,31 @@
 version: "2.0"
 
 responses:
-  utter_medicine:
-    - text: "Go to hospital now!"
-
   utter_open_medicine:
     - text: "... gehe zu 'Medizin-Übersicht'"
 
-  utter_request_able_to_go_alone:
-    - text: "Bist du in der Lage die medizinische Hilfsstelle selbstständig zu erreichen?"
+  utter_save_insured:
+    - text: "... saving insurance-status: 'insurancestatus'\n> erkannter Versicherungsstatus: '{insurancestatus}'"
+
+  utter_insurancestatus_back:
+    - text: "> erkannter Versicherungsstatus: '{insurancestatus}'"
 
-  utter_save_able_to_go_alone:
-    - text: "... saving status: 'able to go alone'"
+  utter_ask_emergency:
+    - text: "Handelt sich es um einen Notfall und du brauchst sofort einen Notarzt?"
 
-  utter_request_insured:
+  utter_ask_insurancestatus:
     - text: "Bist du krankenversichert? "
 
-  utter_save_insured:
-    - text: "... saving insurance-status: 'insurancestatus'\n> erkannter Versicherungsstatus: '{insurancestatus}'"
+  utter_answer_medical_assistance:
+    - text: |
+        > Deine erkannten Informationen sind: 
+          - Notfall: {emergency}
+          - Versicherungsstatus: {insurancestatus}
 
-  utter_medicine_adresses_insured_go_alone:
-    - text: "Folgende Adressen wurden für deine Anlgaben \n- <able to walk?> \n- <versicherungsstatus> \n gefunden .... "
+        Folgende Hilfestellen haben wir für deine Anfrage gefunden: 
+           - ...
```

Comment:
Fehlen hier noch die entsprechenden Hilfestellen oder gibt es da keine in dem MockUp?

Reply:
Und ist Hilfestellen das richtige Wort?

Reply:
@MarieConstantin Für Medizin gibt es nur eine Ansicht für "nicht Krankenversichert" für "versichert" und "Notruf" hingegen gibt es keine Ansicht. Erachtest du es für sinnvoll, dann für versichert einfach eine allgemeine Ansicht zu machen ohne Hilfstellen-Angabe und für "nicht krankenversichert" einfach eine Ausgabe in der die Ambulanz Klinikum Nord aufgeführt ist?

Reply:
Wenn ihr das geklärt habt, bitte auf "Resolve conversation" drücken, um das abzuschließen. Danach muss Marie nochmal "re-reviewen" (Oben bei den Reviewern auf das entsprechende Symbol drücken)